tests/deepsparse/image_classification/test_pipelines.py

```diff
@@ -56,7 +56,7 @@ def test_image_classification_pipeline_preprocessing(
         ]
     )
 
-    ic_pipeline = Pipeline.create("image_classification", zoo_stub)
+    ic_pipeline = Pipeline.create("image_classification", model_path=zoo_stub)
```

Comment:
is `model_path=` necessary now? i like the explicit usage in our implementation but just want to make sure we aren't breaking previous usage

Reply:
any argument which isn't `task` should be explicit